OpenRA.Mods.Common/Scripting/Properties/SellableProperties.cs

```diff
@@ -0,0 +1,35 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("General")]
+	public class SellableProperties : ScriptActorProperties, Requires<SellableInfo>
+	{
+		readonly Sellable sellable;
+
+		public SellableProperties(ScriptContext context, Actor self)
+			: base(context, self)
+		{
+			sellable = self.Trait<Sellable>();
```

Comment:
What if we sell many actors at once? Wouldn't that create a tiny lag without caching?
What I actually want to says, now that we already cache it, do we really want to revert that back?